authservice/domain/port/session_repository_interface.go

```diff
@@ -0,0 +1,12 @@
+package port
+
+import "AdsService/authservice/domain/entity"
+
+type SessionRepository interface {
+	InsertSession(s *entity.Session) error
```

Comment:
```suggestion
	InsertSession(ctx context.Context, s entity.Session) error
```
Используем везде где есть запросы в базу и http.
И читаем почему это важно!